srml/babe/src/lib.rs

```diff
@@ -135,22 +135,13 @@ decl_storage! {
 		/// Current epoch authorities.
 		pub Authorities get(authorities): Vec<(AuthorityId, BabeAuthorityWeight)>;
 
-		/// Slot at which the current epoch started. It is possible that no
-		/// block was authored at the given slot and the epoch change was
-		/// signalled later than this.
-		pub EpochStartSlot get(epoch_start_slot): u64;
+		/// The slot at which the first epoch actually started. This is 0
+		/// until the first block of the chain.
+		pub GenesisSlot get(genesis_slot): u64;
 
 		/// Current slot number.
 		pub CurrentSlot get(current_slot): u64;
 
-		/// Whether secondary slots are enabled in case the VRF-based slot is
```

Comment:
I am OK with removing support for dynamically enabling/disabling secondary slots since we can't change the slot duration anyway, so in practice this is not so useful.

Reply:
Right, that was my rationale. You'd never want to enable/disable secondary slots without tweaking the duration or the `c` parameter